src/megaclient.cpp

```diff
@@ -919,26 +919,58 @@ void MegaClient::activateoverquota(dstime timeleft)
                     t->state = TRANSFERSTATE_RETRYING;
                     t->slot->retrybt.backoff(timeleft);
                     t->slot->retrying = true;
-                    app->transfer_failed(t, API_EOVERQUOTA, timeleft);
+                    app->transfer_failed(t, API_EOVERQUOTA, timeleft, targetHandle);
```

Comment:
This block is for bandwidth overquota, not for storage. Hence, only for downloads, not uploads.
Do we really need to handle the target handle here? Note that it stores the handle of the node that is being downloaded. I think that whether it's foreign or private is irrelevant here.